game-core/src/main/java/games/strategy/engine/framework/startup/ui/InGameLobbyWatcher.java

```diff
@@ -114,11 +114,12 @@ private InGameLobbyWatcher(
         .comment(System.getProperty(LOBBY_GAME_COMMENTS))
         .passworded(passworded)
         .gameVersion("0")
-        .build());
+        .build();
     final ILobbyGameController controller =
         (ILobbyGameController) this.remoteMessenger.getRemote(ILobbyGameController.REMOTE_NAME);
-    controller.postGame(gameId, gameDescription.get());
-    messengerErrorListener = e -> shutDown();
+    synchronized (postMutex) {
```

Comment:
Is the mutex being used to protect GameDescription once again?

It should not be necessary if gameDescription is immutable, immutable objects are thread safe.